sound/soc/sof/intel/hda-dai.c

```diff
@@ -400,6 +400,101 @@ static const struct snd_soc_dai_ops hda_link_dai_ops = {
 	.prepare = hda_link_pcm_prepare,
 };
 
+static struct sof_ipc_dai_config* ssp_dai_config_control(struct snd_sof_dev *sdev,
```

Comment:
@juimonen can we split this patch into smaller ones please. First add the BE hw_params for SSP DAIs. It wont do anything but just send the default hw_config. In the following patch you can add the logic to match the DAI_CONFIG with the runtime hw_params and send that one. In the last patch (if needed) you can add the logic for kcontrol.

And oh btw, the be_dai_link_fixup() function needs to be picked as well no?

Reply:
ok, chopped the last commit into 2 separate. don't get the comment on be_dai_link_fixup?